functions/go/inflate-helm-chart/third_party/sigs.k8s.io/kustomize/api/builtins/HelmChartInflationGenerator.go

```diff
@@ -196,16 +174,32 @@ func (p *HelmChartInflationGeneratorPlugin) replaceValuesInline() error {
 }
 
 // copyValuesFile to avoid branching.  TODO: get rid of this.
-func (p *HelmChartInflationGeneratorPlugin) copyValuesFile() (string, error) {
-	b, err := p.h.Loader().Load(p.ValuesFile)
+func (p *Plugin) copyValuesFile() (string, error) {
+	var b []byte
+	var err error
+	b, err = ioutil.ReadFile(p.ValuesFile)
```

Comment:
If it's a local file, does it require file scheme (e.g. file:///tmp/my-values.yaml)?

Reply:
no, it can be a relative path, e.g. ./tmp/my-values.yaml. I added examples to the table in the README per your comment below